web/src/features/charts/bar-breakdown/EmptyBarBreakdownChart.tsx

```diff
@@ -12,45 +13,34 @@ import { getDataBlockPositions } from './utils';
 interface EmptyBarBreakdownChartProps {
   height: number;
   width: number;
-  isMobile?: boolean;
+  isMobile: boolean;
   overLayText?: string;
 }
 
+const MAX_CO2EQ = 10;
+const MIN_CO2EQ = -1;
+
+const formatTick = (t: number) =>
+  // TODO: format tick depending on displayByEmissions
+  `${t} ${PowerUnits.GIGAWATTS}`;
+
 function EmptyBarBreakdownChart({
   height,
   isMobile,
   overLayText,
   width,
 }: EmptyBarBreakdownChartProps) {
-  const productionData = modeOrderBarBreakdown.map((d) => ({
-    mode: d,
-    gCo2eq: 0,
-    gCo2eqByFuel: {},
-    gCo2eqByFuelAndSource: {},
-    isStorage: false,
-  }));
   const { productionY } = getDataBlockPositions(0, []);
 
-  const maxCO2eqExport = 1;
-  const maxCO2eqImport = 10;
-  const maxCO2eqProduction = 10;
-
   // in CO₂eq
   const co2Scale = useMemo(
     () =>
       scaleLinear()
-        .domain([
-          -maxCO2eqExport || 0,
-          Math.max(maxCO2eqProduction || 0, maxCO2eqImport || 0),
-        ])
+        .domain([MIN_CO2EQ, MAX_CO2EQ])
         .range([0, width - LABEL_MAX_WIDTH - PADDING_X]),
-    [maxCO2eqExport, maxCO2eqProduction, maxCO2eqImport, width]
+    [width]
   );
 
```

Comment:
Oh, interesting with the fancy dynamic calculations that are using fixed values 😂

Reply:
I think we just copied it over and never bothered to change it...

Reply:
Heck I'm not even sure this is used anymore...